packages/discord.js-utilities/src/lib/MessagePrompter.ts

```diff
@@ -0,0 +1,485 @@
+import type {
+	Message,
+	User,
+	TextChannel,
+	NewsChannel,
+	APIMessageContentResolvable,
+	MessageOptions,
+	MessageAdditions,
+	DMChannel,
+	CollectorFilter,
+	EmojiIdentifierResolvable,
+	MessageReaction,
+	EmojiResolvable,
+	GuildEmoji,
+	ReactionEmoji
+} from 'discord.js';
+
+type Constructor<T> = new (...args: any[]) => T;
+type Awaited<T> = PromiseLike<T> | T;
```

Comment:
```suggestion
type Awaited<T> = PromiseLike<T> | T;
type NumberRange = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
```

Reply:
Why would I need a number range? If a user `numberEmojis` to have more than 10 numbers, they could go as far as they want themselves